src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs

```diff
@@ -8,11 +8,14 @@
 namespace System.Collections.Generic
 {
     /// <summary>
-    /// Represents a data structure in which each element has an associated priority
-    /// that determines the order in which the pair is dequeued.
+    ///  Represents a min priority queue.
```

Comment:
Is this three space indent the intended formatting?  We generally don't use that elsewhere to my knowledge.

Reply:
Not sure what the standard formatting is TBH. I expect that @carlossanlop's tool will eventually force a uniform style as we backport content from dotnet-api-docs.